back/app/services/gotenberg_client.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+class GotenbergClient
+  FileTypeUnsupportedError = Class.new(StandardError)
```

Comment:
Any reason not to use the regular syntax?
```suggestion
  class FileTypeUnsupportedError < StandardError; end
```